ext/ldap/tests/gh18902.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+GH-17704 (ldap_search fails when $attributes contains a non-packed array with numerical keys)
+--EXTENSIONS--
+ldap
+--FILE--
+<?php
+$conn = ldap_connect();
+try {
+	ldap_exop($conn,null);
+} catch (\ValueError $e) {
+	echo $e->getMessage(), PHP_EOL;
+}
```

Comment:
Remove this case, as it is covered by the empty string one, and this reduces future churn when passing null becomes a TypeError.